Bug: unknown type error with compiling with gcc 13.3

When building the benchmark on a system with gcc 13.3.0, I get the following error 

mpicxx -std=c++11 -g -O2 -march=native -DNOGEOPM   -c bench.cpp -o bench.o 
In file included from bench.cpp:51:
./bench.hpp:69:5: error: unknown type name 'intmax_t'; did you mean '__intmax_t'?
   69 |     intmax_t start_time;
      |     ^~~~~~~~
      |     __intmax_t

Possible fix:
This is because `intmax_t` is installed in `cstdint.h`, which is not included in bench.hpp

--- a/bench.hpp
+++ b/bench.hpp
@@ -54,6 +54,7 @@
 #include <cstddef>
 #include <vector>
 #include <iostream>
+#include <cstdint>
 
 struct BenchmarkConfig final
 {

